queuebot/cogs/queue/suggestion.py

```diff
@@ -112,7 +121,7 @@ def status(self):
         """, self.record['idx'])
         await self.update_inplace()
 
-    async def move_to_public_queue(self):
+    async def move_to_public_queue(self, who=None, reason=None):
```

Comment:
Honestly, I think these should be kwarg-only, at minimum the `reason`, just to match discord.py@rewrite's methods.